CHANGELOG.md

```diff
@@ -31,7 +35,7 @@ All notable changes to this project will be documented in this file.
 - Cleaned up argument passing and other repetitive code using dictionary and set comparisons for easy maintenance. (Converted variable names to a consistent snake_case from a mix of kebab-case and snake_case)
 - Fixed change history table processing to allow mixed case names when '"' are used in the name.
 - Moved most error, log and warning messages and query strings to global or class variables.
-- Updated readme to cover new authentication methods 
+- Updated readme to cover new authentication methods
```

Comment:
Please rebase from `CBRO-1218-undo-migrations`,
the top commit from it already removes trailing whitespaces and tabs https://github.com/salemove/schemachange/pull/4/commits/94deeb5947f091d038fe5d4ea62e5e814059fdfc.

It will be easier to review this PR if it has your changes only.
Also feels unnecessary to write a bunch of logic in `recalculate_checksum_command` only to split it in the next commit, why not to split it initially?

Reply:
> why not to split it initially
One was feature, other was refactoring

Reply:
anyways, it is rebased. I'll still point to master. You can skip first 3 commits

Reply:
thank you